### PR TITLE
Corrected the example given

### PR DESCRIPTION
**Pull request by ross-bohr**
The example is specifying the RG name in Azure as "example" but in the shell command example given the RG name is "group1" thus this change is correcting the HCL script to match the name "group1".

### CODE REVIEW

File: website/docs/r/resource_group.html.markdown [hunk 1]
```diff
@@ -14,7 +14,7 @@ Manages a Resource Group.
 
 ```hcl
 resource "azurerm_resource_group" "example" {
-  name     = "example"
+  name     = "group1"
```
Comment (katbyte) [inline website/docs/r/resource_group.html.markdown:17]:
```suggestion
  name     = "example"
```

**Review by katbyte — CHANGES_REQUESTED** [review]
Could we keep the resource group name as example and change the import statement? thanks!

**Review by katbyte — APPROVED** [review]
Hey @ross-bohr  - hope you don't mind but i pushed the changes to get this merged today. Thanks! LGTM 👍

**Comment by ross-bohr** [comment]
@katbyte No worries, it's good now, thanks! :)

**Pull request merged**